global/scripts/MyelinMap_BC.sh

```diff
@@ -66,79 +57,84 @@ AtlasSpaceFolder=${SubjFolder}/MNINonLinear
 log_Msg "AtlasSpaceFolder: $AtlasSpaceFolder"
 T1wFolder=${SubjFolder}/T1w
 log_Msg "T1wFolder: $T1wFolder"
+NativeFolder=${AtlasSpaceFolder}/Native
+log_Msg "NativeFolder: $NativeFolder"
+NativeT1wFolder=${T1wFolder}/Native
+log_Msg "NativeT1wFolder: $NativeT1wFolder"
 
-if [[ "$InputMesh" == "$OutputMesh" ]]; then
-	log_Err_Abort "the input mesh ${InputMesh} and the output mesh ${OutputMesh} shouldn't be the same resolution!"
-fi
-
-case "$InputMesh" in
-	(native)
-		InputMeshString="native"
-		InputFolder=${AtlasSpaceFolder}/Native
-		InputT1wFolder=${T1wFolder}/Native
-		InputBCMapName="MyelinMap"
-		;;
+case "$LowResMesh" in
 	(164)
-		InputMeshString=164k_fs_LR
-		InputFolder=${AtlasSpaceFolder}
-		InputT1wFolder=${AtlasSpaceFolder}
-		InputBCMapName="MyelinMap"
+		LowResMeshString=164k_fs_LR
+		LowResFolder=${AtlasSpaceFolder}
+		LowResT1wFolder=${AtlasSpaceFolder}
+		LowResBCMapName="MyelinMap"
 		;;
 	(*)
-		if [[ $InputMesh =~ ^[0-9]+$ ]]; then
-			InputMeshString=${InputMesh}k_fs_LR
-			InputFolder=${AtlasSpaceFolder}/fsaverage_LR${InputMesh}k
-			InputT1wFolder=${T1wFolder}/fsaverage_LR${InputMesh}k
-			InputBCMapName="atlas_MyelinMap"
+		if [[ $LowResMesh =~ ^[0-9]+$ ]]; then
+			LowResMeshString=${LowResMesh}k_fs_LR
+			LowResFolder=${AtlasSpaceFolder}/fsaverage_LR${LowResMesh}k
+			LowResT1wFolder=${T1wFolder}/fsaverage_LR${LowResMesh}k
+			LowResBCMapName="atlas_MyelinMap"
 		else
-        	log_Err_Abort "unrecognized --input-mesh value '$InputMesh', valid options are 'native' or numbers like '32', '164', etc."
+        	log_Err_Abort "unrecognized --low-res-mesh value '$LowResMesh', valid options are numbers like '32', '164', etc."
 		fi
         ;;
 esac
 
-case "$OutputMesh" in
-	(native)
-		OutputMeshString="native"
-		OutputFolder=${AtlasSpaceFolder}/Native
-		OutputT1wFolder=${T1wFolder}/Native
-		OutputBCMapName="MyelinMap"
+case "$RegName" in
+	("")
```

Comment:
empty string should probably be taken to mean the same as MSMSulc, these settings look like they would fail (it doesn't make sense to resample the group template to individual native mesh without using a registration).

Reply:
I see. I have changed it in the latest commit.